website_product_attribute_description/static/src/js/tooltip.js

```diff
@@ -0,0 +1,15 @@
+odoo.define("product_attribute_description.product_attribute_description", function () {
```

Comment:
Can you tighten up the selectors here, this is prone to conflict with other modules.